assets/css/main.scss

```diff
@@ -1,4 +1,5 @@
 ---
+redirect_to: false
```

Comment:
Had to add `redirect_to: false` otherwise the generated CSS bundle was empty 🤷‍♂️